security.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Eiffel-community Vulnerability and security Reports</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="./images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="./manifest.json">
+    <link rel="mask-icon" href="./images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel='stylesheet' href='./css/index.css'>
+    <meta name="theme-color" content="#ffffff">
+		<meta charset="UTF-8">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <a class="header-logo no-underline-hover" href="/">
+          <img src="images/eiffel-logo.png" class="header-logo-image"/>
+        </a>
+
+        <nav class="header-nav">
+          <a class="header-footer-nav-link" href="https://github.com/eiffel-community/eiffel">Protocol</a>
+          <a class="header-footer-nav-link" href="/community.html">Community</a>
+          <a class="header-footer-nav-link" href="/summit.html">Eiffel Summit</a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://www.youtube.com/channel/UC3Zx84ecQ1jyGSc9709xrZg">
+            <img src="images/youtube-channel-icon.png" class="header-nav-link-image"/>
+          </a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://github.com/eiffel-community">
+            <img src="images/github-organization-icon.png" class="header-nav-link-image"/>
+          </a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://groups.google.com/forum/#!forum/eiffel-community">
+            <img src="images/google-group-icon.png" class="header-nav-link-image"/>
+          </a>
+        </nav>
+      </div>
+    </header>
+    <section>
+      <div class="container">
+        <p class="splash-text-large">Eiffel-community vulnerability and security reports</p>
+        <p class="splash-text-small">
+          Publicly disclosed security vulnerabilities and fixes
```

Comment:
Subtitles should also be capitalized according to https://www.constant-content.com/content-writing-service/2008/02/subtitles-should-be-capitalized-just-like-titles/